### DISCUSSION TITLE
如何找到最便宜的機票

### DISCUSSION THREAD

**Discussion by Elaine4488** [1101-GS4719-C/Python-Community-GS4719-C#3679] [General]
很多人會趁長假的時候安排小旅行，但要如何找到最划算的機票呢？像我的話，通常會直接用Google搜尋兩地的航班資訊，比較價格區間來選擇。不過如果有時間，在中途多停一個站點，其實有時候可以大幅降低機票價格。

之前還聽說過，有人因為對這個需求感興趣，乾脆寫了一個專門找便宜機票的程式，最後甚至靠這個創業，幫大家客製化尋找最低價的航班。你們都怎麼找機票的呢？

**Comment by 113607514** [comment]
每次訂機票都像是一場尋寶冒險。記得有一次，我只是隨手打開Google Flights，沒想到竟然找到了一張比平常便宜快一半的票，目的地還是我一直想去的東京。那一刻，真的有種中了樂透的感覺。  

我還試過凌晨三點不睡覺，硬是刷了半個小時的比價網站，最後才發現，換個城市進出就能省下一頓大餐的錢。雖然當下覺得自己是不是太斤斤計較，但當飛機降落，看著窗外那些熟悉又陌生的街景時，一切都值得了。  

或許這就是旅行的魅力吧。不是每一次都能撿到超便宜的票，但每一次的搜尋，都是為了離想去的地方更近一步。

**Comment by ShuliANN** [comment]
有聽說在查找機票時要開無痕，也許是不要讓網站抓到你有想找這個機票，進而去漲價。
也有一個民間傳說是在禮拜二時買會比較便宜。

**Comment by yuan3x2** [comment]
買機票要看去回程的日期，在附近的時間都點點看，通常星期六的機票都會比較貴，星期一二的比較便宜，可以用這種方式找到更便宜的機票！

**Comment by Taoyi-Chiang** [comment]
Google Flight 可以追蹤機票，我覺得這個功能滿好的，可以隨時報價，還會幫你列出跟歷史票價做比較。
這樣就不用隨時看票價，應該是滿貼心的設計。
